lib/internal/Magento/Framework/Data/Form/Element/Editor.php

```diff
@@ -286,15 +286,19 @@ protected function _getPluginButtonsHtml($visible = true)
 
         // Button to media images insertion window
         if ($this->getConfig('add_images')) {
+            $mediaGallery = (null !== $this->getConfig('is_new_media_gallery')) ?
+                          $this->getConfig('is_new_media_gallery') : 0;
             $buttonsHtml .= $this->_getButtonHtml(
                 [
                     'title' => $this->translate('Insert Image...'),
                     'onclick' => "MediabrowserUtility.openDialog('"
                         . $this->getConfig('files_browser_window_url')
                         . "target_element_id/" . $this->getHtmlId() . "/"
                         . (null !== $this->getConfig('store_id') ? 'store/'
-                            . $this->getConfig('store_id') . '/' : '')
-                        . "')",
+                        . $this->getConfig('store_id') . '/' : '')
+                        . "', null, null, null, { is_new_media_gallery:"
+                        . " $mediaGallery"
```

Comment:
It's not clear why `$mediaGallery` is inserted into `is_new_media_gallery`, could be better to rename variable into `$isNewMediaGallery`